lib/oelite/fetch/git.py

```diff
@@ -267,6 +267,17 @@ def unpack(self, d):
         return True
 
     def mirror(self, mirror=os.getcwd()):
+        if self.is_local:
+            # Don't mirror if the source is local. Concept taken from fetch
+            # Do check revision exists before accepting though
+            print "Warning: Skipping creating mirror for local path %s"%(self.uri)
+            repo = oelite.git.GitRepository(self.repo)
+            if self.has_rev(repo):
```

Comment:
The mirror task is run after fetch, so I don't think this duplication of code makes a lot of sense.  It doesn't hurt a lot either, just a slightly confusing, as it kind of hints that it makes sense to check for it here.  So yes, I prefer to get rid of this code here :)